src/test/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultControllerTestSuite.scala

```diff
@@ -690,6 +693,24 @@ class CloudStackVaultControllerTestSuite extends FlatSpec with BaseTestSuite wit
     assert(revokedTokenList == checkedRevokedTokens)
   }
 
+  "createMissingAccountTokenTags" should "trow IllegalArgumentException if tag key does not in (VaultRO, VaultRW)" in {
```

Comment:
"createMissingAccountTokenTags" should "throw IllegalArgumentException if tag key is not one of (VaultRO, VaultRW)"

Reply:
ok